manifests/server/puppetserver.pp

```diff
@@ -240,7 +240,7 @@
   }
 
   unless $facts['os']['family'] == 'FreeBSD' or
-    ($facts['os']['family'] == 'Debian' and $puppet::params::aio_package) {
+  ($facts['os']['family'] == 'Debian' and $puppet::params::aio_package) {
```

Comment:
I consider this a bug in the indent rule. Not going to dive into it in the weekend, just want to let you know.

Reply:
OK, just let me know if you want me to remove that commit from the branch.

Reply:
In https://github.com/voxpupuli/puppet-lint-strict_indent-check/pull/28 I already added examples for this, though I suspect that in this case you should use braces around the statements.

Reply:
I'm not sure where you're suggesting I use braces (`{}`) in there?